dev/com.ibm.ws.st.docker.base/feature.xml

```diff
@@ -18,7 +18,7 @@
    </license>
 
    <requires>
-      <import feature="com.ibm.ws.st.liberty.tools.base" version="0.0.1.qualifier" match="greaterOrEqual"/>
+      <import feature="com.ibm.ws.st.liberty.tools.base" version="1.0.2.qualifier" match="greaterOrEqual"/>
```

Comment:
Remove .qualifier here as it is not needed.

Reply:
Fixed.